src/vaadin-combo-box-mixin.html

```diff
@@ -262,6 +262,17 @@
       this._observer = new Polymer.FlattenedNodesObserver(this, info => {
         this._setTemplateFromNodes(info.addedNodes);
       });
+
+      const bringToFrontListener = (e) => {
+        requestAnimationFrame(() => {
+          if (this.$.overlay.$.dropdown.$.overlay.bringToFront) {
```

Comment:
I would double check that shadow roots do exist:

```js
const overlay = this.$.overlay;
const dropdown = overlay && overlay.$.dropdown;
if (dropdown && dropdown.$.overlay && dropdown.$.overlay.bringToFront) 
```

Reply:
True, relying on requestAnimationFrame timing is actually dangerous. Note that you might also need to check `dropdown.$`

Reply:
done as we discussed